cluster/manifests/skipper/deployment.yaml

```
@@ -557,6 +557,9 @@ spec:
           protocol: TCP
         args:
           - "routesrv"
+{{ if eq .Cluster.ConfigItems.skipper_eks_migration_enabled "true" }}
+          - "-forward-backend-url=http://skipper-ingress-eks.{{ .Cluster.Alias }}.zalan.do"
```

```suggestion
          - "-forward-backend-url=http://skipper-ingress-eks.{{ .Values.hosted_zone }}"
```

Using the `hosted_zone` variable provides the right account level hosted zone. `Alias` would only work for the non-EKS cluster.